mkt/lookup/templates/lookup/includes/app_search.html

```diff
@@ -5,6 +5,7 @@
       {{ _('App Lookup') }}
     </label>
     <input type="search" id="app-search" name="q" autocomplete="off">
+    <input type="hidden" id="app-lang" name="lang" autocomplete="off" value="en-US">
```

Comment:
@diox If you know, can you advise me on how to use the user's locale here (instead of this hard-coded `'en-US'`)?